<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSlider" name="horizontalSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="myGraphicsView" name="graphicsView">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QTabWidget" name="toolsOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Narzędzia</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QSlider" name="thresholdSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>172</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
    border: 1px solid black;
    height: 20px;
    margin: 2px 0;
}

QSlider::handle:horizontal {
	background-color: black;
    border: 1px solid black;
    width: 2px;
    border-radius: 3px;
}

QSlider::sub-page {
	background: #3add36;
	border: 1px solid black;
}

QSlider::sub-page:horizontal {
    border: 1px solid black;
    height: 20px;
    margin: 2px 0;
}</string>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="thresholdLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="featherLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>21</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QSlider" name="featherSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>172</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
    border: 1px solid black;
    height: 20px;
    margin: 2px 0;
}

QSlider::handle:horizontal {
	background-color: black;
    border: 1px solid black;
    width: 2px;
    border-radius: 3px;
}

QSlider::sub-page {
	background: #3add36;
	border: 1px solid black;
}

QSlider::sub-page:horizontal {
    border: 1px solid black;
    height: 20px;
    margin: 2px 0;
}</string>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Progowanie</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="featherCheckBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Zmiękczanie krawędzi</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="selectByColorButton">
         <property name="text">
          <string>Magiczna Różdzka</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="rectangleSelectButton">
         <property name="text">
          <string>Zaznaczenie Prostokątne</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuFiltry">
    <property name="title">
     <string>Filtry</string>
    </property>
    <widget class="QMenu" name="menuWykrycie_kraw_dzi">
     <property name="title">
      <string>Wykrycie krawędzi</string>
     </property>
     <addaction name="actionLaplace"/>
    </widget>
    <widget class="QMenu" name="menuRozmycie">
     <property name="title">
      <string>Rozmycie</string>
     </property>
     <addaction name="actionLowPass"/>
     <addaction name="actionGauss"/>
    </widget>
    <widget class="QMenu" name="menuUwydatnienie">
     <property name="title">
      <string>Uwydatnienie</string>
     </property>
     <addaction name="actionHighPass"/>
    </widget>
    <addaction name="menuRozmycie"/>
    <addaction name="menuUwydatnienie"/>
    <addaction name="menuWykrycie_kraw_dzi"/>
    <addaction name="actionCustomMask"/>
   </widget>
   <widget class="QMenu" name="menuKolory">
    <property name="title">
     <string>Kolory</string>
    </property>
    <addaction name="actionHueSaturation"/>
    <addaction name="actionColorize"/>
    <addaction name="actionBrightnessContrast"/>
    <addaction name="separator"/>
    <addaction name="actionNegative"/>
    <addaction name="actionSepia"/>
    <addaction name="actionDesaturate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFiltry"/>
   <addaction name="menuKolory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionCustomMask">
   <property name="text">
    <string>Własna maska</string>
   </property>
  </action>
  <action name="actionLaplace">
   <property name="text">
    <string>Laplace</string>
   </property>
  </action>
  <action name="actionLowPass">
   <property name="text">
    <string>Uśredniający</string>
   </property>
  </action>
  <action name="actionGauss">
   <property name="text">
    <string>Gauss</string>
   </property>
  </action>
  <action name="actionHighPass">
   <property name="text">
    <string>Wyostrzenie</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Zapisz jako...</string>
   </property>
  </action>
  <action name="actionHueSaturation">
   <property name="text">
    <string>Barwa i nasycenie</string>
   </property>
  </action>
  <action name="actionColorize">
   <property name="text">
    <string>Koloryzacja</string>
   </property>
  </action>
  <action name="actionBrightnessContrast">
   <property name="text">
    <string>Jasność i kontrast</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negatyw</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="text">
    <string>Sepia</string>
   </property>
  </action>
  <action name="actionDesaturate">
   <property name="text">
    <string>Desaturacja</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
